### DISCUSSION TITLE
Delete suffix from Title

### DISCUSSION THREAD

**Discussion by AlexUA1983** [facebook/docusaurus#7491] [Ideas]
Hi
I'm read this [https://github.com/facebook/docusaurus/discussions/6640](url)

wrapping
```
import React from 'react';
import Layout from '@theme-original/Layout';
import Head from '@docusaurus/Head';

export default function LayoutWrapper(props) {
    return (
        <>
            <Layout {...props} />
            <Head titleTemplate = "%s">
                {props.title && <title>{props.title}</title>}
            </Head>
        </>
    );
}
```

But this don't work :(

Please make it possible in the main config (docusaurus.config.js) to disable this suffix 🙏🙏🙏

**Comment by slorber** [comment]
Let's track this as part of issue https://github.com/facebook/docusaurus/issues/5878, no need to open another discussion

**Reply by slorber** [reply]
It will be possible in v3.8 soon: https://github.com/facebook/docusaurus/pull/11090
